apps/web-client/src/lib/components/supplier-orders/OrderedTable.svelte

```diff
@@ -81,7 +84,8 @@
 							{new Date(created).toLocaleString()}
 						</span>
 					</td>
-					<td class="text-right">
+					<td class="flex items-center justify-evenly text-right">
```

Comment:
The table needs an extra `<td scope="col">`  in the table head

Reply:
I can't quite place where this is missing..

Reply:
In `thead`